testgrid/config.yaml

```diff
@@ -2447,8 +2447,8 @@ test_groups:
 - name: pull-cluster-api-test
   gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-cluster-api-test
   num_columns_recent: 20
-- name: pull-cluster-api-provider-aws-test-creds
-  gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-cluster-api-provider-aws-test-creds
+- name: cluster-api-provider-aws-test-creds
```

Comment:
Consider: Often periodic jobs start with ci

Reply:
Will integrate changes requested, thanks.

Reply:
There are a lot of pseudo and unspoken conventions. Sorry.